core/AbstractProfile.php

```diff
@@ -254,7 +254,7 @@ public static function significantChanges($old, $new)
      * 
      * @return array of arrays of the form [['level' => $level, 'explanation' => $explanation, 'reason' => $reason]];
      */
-    public function openroamingRedinessTest() {
+    public function openroamingReadinessTest() {
```

Comment:
Is this function part of an API that cannot be changed without breaking compatibility?